Automattic/woocommerce-payments
# Fix encoded entities in the store name in the WooPay preview section

#### Changes proposed in this Pull Request

Decode entities for the store name in the WooPay preview section.

#### Testing instructions

* On the merchant store, go to WP Admin -> Settings -> General
* Update the site title and include some HTML entities, like `%#"&`, and save
* Go to Payments -> Settings -> Express checkouts -> WooPay -> Customize
* Under Checkout appearance, confirm the store name displays the entities decoded. `&` should show up as `&`, and not as `&amp;`

<img width="1747" alt="image" src="https://user-images.githubusercontent.com/41606954/211825222-d5fe923a-3a1f-47a1-897b-8e54a12a964d.png">

-------------------


**Post merge**
